applications/asset_tracker/src/gps_controller/gps_controller.c

```diff
@@ -155,8 +155,8 @@ int gps_control_init(struct k_work_q *work_q, gps_event_handler_t handler)
 		return -EINVAL;
 	}
 
-	if (IS_ENABLED(CONFIG_AGPS_SINGLE_CELL_ONLY)) {
-		LOG_INF("Cell-based location enabled, skipping GPS init");
+	if (!IS_ENABLED(CONFIG_NRF9160_GPS) && !IS_ENABLED(CONFIG_GPS_SIM)) {
```

Comment:
`gps_control_init` initializes `start_work` and `stop_work`. What happens if the `main.c` tries to call either `gps_control_start` or `gps_control_stop` that tries to schedule un-initialized work items.

This could be solved by making `is_init` global and checking if the library is in fact initialized before scheduling the work items internally in `gps_control_start/stop`.

Reply:
The new logic should be in another PR I think.
Here we just adapt to cellular positioning change.
@jayteemo please comment.

Reply:
Seems out of scope for this PR.
Previous behavior when single cell only was enabled was to exit early.  As Zou mentioned, perhaps we need to file an issue for asset tracker gps behavior now that cell_pos is separate?

Reply:
Sounds reasonable, the second comment was more of an observation. The condition of returning 0 has changed. At least it isn't obvious that the new condition is apples-to-apples- comparable to the previous, if that is the case. That's why I raised the concern of the application behavior being persistent between these changes.